demo-app/frontend/src/App.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState, useEffect } from 'react';
+import './App.css';
+
+function App() {
+  const [message, setMessage] = useState('');
+
+  useEffect(() => {
+    fetch('http://localhost:3002/api/data')
```

Comment:
I'd recommend either setting up the [react dev proxy](https://create-react-app.dev/docs/proxying-api-requests-in-development/#configuring-the-proxy-manually) or putting the demo app behind nginx so we don't have to 
1. manually define the host/port, or
2. set up CORS on the server